### PR TITLE
Migrate to remarkjs

### PR DESCRIPTION
**Pull request by wooorm**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,10 +4,13 @@
 
 [![npm status](http://img.shields.io/npm/v/remark-git-contributors.svg)](https://www.npmjs.org/package/remark-git-contributors)
 [![node](https://img.shields.io/node/v/remark-git-contributors.svg)](https://www.npmjs.org/package/remark-git-contributors)
-[![Travis build status](https://img.shields.io/travis/vweevers/remark-git-contributors.svg?label=travis)](http://travis-ci.org/vweevers/remark-git-contributors)
-[![AppVeyor build status](https://img.shields.io/appveyor/ci/vweevers/remark-git-contributors.svg?label=appveyor)](https://ci.appveyor.com/project/vweevers/remark-git-contributors)
-[![Dependency status](https://img.shields.io/david/vweevers/remark-git-contributors.svg)](https://david-dm.org/vweevers/remark-git-contributors)
+[![Travis build status](https://img.shields.io/travis/remarkjs/remark-git-contributors.svg?label=travis)](http://travis-ci.org/remarkjs/remark-git-contributors)
+[![AppVeyor build status](https://img.shields.io/appveyor/ci/remarkjs/remark-git-contributors.svg?label=appveyor)](https://ci.appveyor.com/project/remarkjs/remark-git-contributors)
```
Comment (vweevers) [inline readme.md:8]: This one shows as:

image

Comment (wooorm) [reply readme.md:8]: Weird. I added it but it’s stuck on https://ci.appveyor.com/project/wooorm/remark-git-contributors now... How does one fix this 🤔

Comment (vweevers) [reply readme.md:8]: Do you have an organizational account on AppVeyor? See https://www.appveyor.com/docs/team-setup/#setting-up-appveyor-account-for-github-organization

Comment (wooorm) [reply readme.md:8]: Nice, found it, setting it up!

**Review by vweevers — APPROVED** [review]
Good to go once AppVeyor is set up

File: readme.md (hunk 1 above)
Comment (wooorm) [reply readme.md:8]: It’s there!

**Pull request merged**